addons/medical_statemachine/initSettings.sqf

```diff
@@ -41,3 +41,13 @@
     [1, 3600, 30],
     true
 ] call CBA_settings_fnc_init;
+
+[
+    QGVAR(epiBoostsSpontaneousWakeUp), "SLIDER",
+    [LSTRING(epiBoostsSpontaneousWakeUpDisplayName), LSTRING(epiBoostsSpontaneousWakeUp_Description)],
+    _categoryArray,
+    [1,15,1,1], // [min, max, default value, trailing decimals (-1 for whole numbers only)]
```

Comment:
Is the max value tied to `SPONTANEOUS_WAKE_UP_INTERVAL`?

Reply:
It boosts by increasing how often it's checked
`SPONTANEOUS_WAKE_UP_INTERVAL (15 sec) / GVAR(epiBoostsSpontaneousWakeUp)`
So setting range could really be anything, I increased max to 30x